scripts/build_azuresdk.sh

```diff
@@ -68,18 +68,36 @@ azure_configure_opts+=(
     "-DOPENSSL_INCLUDE_DIR=$DEPENDENCY_DIR/openssl/include"
     "-DOPENSSL_CRYPTO_LIBRARY=$DEPENDENCY_DIR/openssl/lib/libcrypto.a"
     "-DOPENSSL_SSL_LIBRARY=$DEPENDENCY_DIR/openssl/lib/libssl.a"
-    "-DUUID_INCLUDE_DIR=$DEPENDENCY_DIR/uuid/include"
-    "-DUUID_LIBRARIES=$DEPENDENCY_DIR/uuid/lib/libuuid.a"
     "-DCURL_INCLUDE_DIRS=$DEPENDENCY_DIR/curl/include"
     "-DCURL_LIBRARIES=$DEPENDENCY_DIR/curl/lib/libcurl.a"
     "-DEXTRA_INCLUDE=$DEPENDENCY_DIR/zlib/include"
 )
 
+if [[ "$PLATFORM" == "linux" ]]; then
```

Comment:
@smtakeda macOS has libuuid by default and including our prebuilt libuuid causes a build error in the Azure Storage Lite SDK. The SDK also recommends that you use the system libuuid for macOS, so I dropped this from macOS.

Reply:
Just like windows. The unreferenced error for gettext can be fixed by removing path to the gettext package on macos. by running  
brew unlink gettext
This will mark gettext as unavailable and we do not see the unreferenced error.
If libuuid is available on macos that should be fine.

Reply:
+1